packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/Examples.6/ColumnHighlightGreen.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { FlowsheetDataGrid } from 'terra-data-grid';
+
+const ColumnHighlightGreen = () => {
+  const gridDataJSON = {
+    cols: [
+      { id: 'Column-0', displayName: 'Vitals' },
+      { id: 'Column-1', displayName: 'March 16' },
+      {
+        id: 'Column-2',
+        displayName: 'March 17',
+        columnHighlightColor: 'green',
```

Comment:
Use the constants here and also update all other instances in the examples and tests.

Reply:
Changed in https://github.com/cerner/terra-framework/pull/1985/commits/c47cc862bbd19e065b0ba5145e413154f609d94b